test/framework/src/main/java/org/opensearch/test/ParameterizedOpenSearchIntegTestCase.java

```diff
@@ -48,7 +48,7 @@ public void afterTests() {
     }
 
     public void indexRandomForConcurrentSearch(String... indices) throws InterruptedException {
-        if (dynamicSettings.get(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey()).equals("true")) {
+        if ("true".equals(dynamicSettings.get(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey()))) {
```

Comment:
Won't this if statement always evaluate to false? `setupSuiteScopeCluster` is called during `beforeClass` but the settings are applied during `beforeTest`

cc @neetikasinghal

Reply:
this function shouldnt be called in setupSuiteScopeCluster, this function is meant to be called only in the tests.

Reply:
+1 @weiwang118 can you revert this change and add a code comment for this method?